eclipse-scout-core/src/menu/Menu.ts

```diff
@@ -731,4 +732,15 @@ export class Menu extends Action implements MenuModel {
     }
     return false;
   }
+
+  protected override _tooltipText(): string {
+    if (this._showTextAsTooltip()) {
+      return strings.join('\n\n', this.text, this.tooltipText);
+    }
+    return super._tooltipText();
+  }
+
+  protected _showTextAsTooltip(): boolean {
```

Comment:
Please add a comment or jsdoc what the purpose of this function is (e..g same as in _renderText )